mule-maven-plugin/src/main/java/org/mule/tools/maven/mojo/ValidateMojo.java

```diff
@@ -36,6 +39,12 @@
 public class ValidateMojo extends AbstractMuleMojo {
 
   private static final String MIN_MAVEN_VERSION = "3.3.3";
+  private static final DependencyValidator TEST_SCOPE_DEPENDENCY_VALIDATOR = new TestScopeDependencyValidator(
```

Comment:
Name should be something with MUnit, TEST_SCOPE_MUNIT_DEPENDENCY_VALIDATOR

Reply:
But what if we add a new dependency in the future? Is just a coincidence that now it only validates an MUnit dependency.